tests/integration/update_cluster/existing_iam/in-v1alpha2.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: kops/v1alpha2
```

Comment:
Could we make this test a master HA test, similar to what we find in https://github.com/kubernetes/kops/blob/master/tests/integration/update_cluster/ha/in-v1alpha1.yaml. This would help catch any future issues changes to this code may cause to HA setups.

Reply:
Good idea, will do.

Reply:
Test updated to use HA masters

Reply:
Looks like you only have 2 masters in the test, that would actually fail as Kops requires you to have a odd number of masters for an HA setup.